librz/core/canalysis.c

```diff
@@ -481,30 +488,39 @@ static int bb_cmp(const void *a, const void *b, void *user) {
 	return ba->addr - bb->addr;
 }
 
-RZ_IPI void rz_core_analysis_bbs_info_print(RzCore *core, RzAnalysisFunction *fcn, RzCmdStateOutput *state) {
-	rz_return_if_fail(core && fcn && state);
+RZ_IPI char *rz_core_analysis_bbs_info_print(RzCore *core, RzAnalysisFunction *fcn, RzCmdStateOutput *state) {
```

Comment:
1. Add ownership for the return
2. Maybe rename it? As it doesn't print anything anymore, `rz_core_analysis_bbs_as_string()` or something like that?